config/settings.yml

```diff
@@ -224,6 +224,8 @@ edu:
       - vishnhav.ashok@va.gov
       - donna.saunders@va.gov
       - ariana.adili@govcio.com
+      - marelby.hernandez@va.gov
+      - nawar.hussein@va.gov
```

Comment:
Can you update the PR description and title to reflect adding two people and not just Marelby

Reply:
Done.  Good catch, Nawar was a late addition and I forgot to add him to the description and title.  Thanks.

Reply:
thanks!